### PR TITLE
Add admin/measures methods

### LINKED ISSUE #321: Implement Admin Measures API
Mastodon's Admin Measures API allows an admin to obtain quantitative metrics about the server. The goal of this task is to implement the following endpoints of this API in BigBone:

- Get measurable data

The Admin Measures API as it is documented by Mastodon can be found here:
https://docs.joinmastodon.org/methods/admin/measures/

Tip: A lot of other endpoints similar to this have already been implemented. In order to get an idea how to implement this, I suggest that you study the code upfront. Start with one of the `*Methods.kt` files.

Please consult the following documents to make sure your contribution meets our projects criteria:

- [Code of Conduct](https://github.com/andregasser/bigbone/blob/master/CODE_OF_CONDUCT.md)
- [Code Contribution Guidelines](https://github.com/andregasser/bigbone/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by PattaFeuFeu**
# Description

Wow, that took way more time than anticipated. I took the ticket with the lowest amount of methods to add (one in this case), thinking it wouldn’t take too long. I should have checked the documentation _first_. 😄😵‍💫

This PR adds the [`measures` API](https://docs.joinmastodon.org/methods/admin/measures/) which is part of the [`admin` scope](https://docs.joinmastodon.org/methods/admin/).

So far, we haven’t done this but I’d like to start:
I have added `admin` packages to both `entity` and `method` to structure this a bit more as we’ll soon get even more admin-scoped methods.

To ensure that required parameters are always added for any given key, I have chosen the `sealed class` approach. This brings a bit of overhead and some duplication but I liked this better than having multiple `if`s or a giant `when` in the method.

Closes #321 

# Type of Change

(Keep the one that applies, remove the rest)

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update
  - [API Coverage](https://github.com/andregasser/bigbone/wiki/Mastodon-API-Coverage) page needs to be updated

# How Has This Been Tested?

New tests added, `gradle check`

# Mandatory Checklist

- [x] My change follows the projects coding style
- [x] I ran `gradle check` and there were no errors reported
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] KDoc added to all public methods

# Optional Things To Check

The items below are some more things to check before asking other people to review your code.

- In case you worked on new features: Did you also implement the reactive endpoint (bigbone-rx)? ✅ 
- In case you added new *Methods classes: Did you also add it to `MastodonClient`? ✅
- In case you worked on endpoints: Please mention in the PR that [API Coverage](https://github.com/andregasser/bigbone/wiki/Mastodon-API-Coverage) page needs to be updated (if needed) ✅

### CODE REVIEW

File: bigbone/src/main/kotlin/social/bigbone/api/method/admin/AdminMeasuresMethods.kt [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+import social.bigbone.Parameters
+import social.bigbone.api.entity.admin.AdminMeasure
+import social.bigbone.api.entity.admin.AdminMeasure.Key
+import java.time.Instant
+
+/**
+ * Obtain quantitative metrics about the server.
+ * @see <a href="https://docs.joinmastodon.org/methods/admin/measures/">Mastodon admin/measures API methods</a>
+ */
+class AdminMeasuresMethods(private val client: MastodonClient) {
```
Comment (andregasser) [inline bigbone/src/main/kotlin/social/bigbone/api/method/admin/AdminMeasuresMethods.kt:14]: To remain consistent, we should rename this class to `AdminMeasureMethods`.

**Review by andregasser — CHANGES_REQUESTED** [review]
Hello @PattaFeuFeu I like the approach taken in this PR. From my side, I see no issues if we continue with the admin endpoints like this. I have just added a comment regarding a class name, but otherwise, it is looking good to me :-)

File: bigbone/src/main/kotlin/social/bigbone/api/method/admin/AdminMeasuresMethods.kt (hunk 1 above)
Comment (PattaFeuFeu) [reply bigbone/src/main/kotlin/social/bigbone/api/method/admin/AdminMeasuresMethods.kt:14]: But it’s the `/measures` API, shouldn’t it be `MeasuresMethods` then? 🤔

Comment (andregasser) [reply bigbone/src/main/kotlin/social/bigbone/api/method/admin/AdminMeasuresMethods.kt:14]: I know, we had long discussions about this and then agreed on using the singular version of the word.

Comment (bocops) [reply bigbone/src/main/kotlin/social/bigbone/api/method/admin/AdminMeasuresMethods.kt:14]: Is this about singular vs. plural, or about the "Admin" prefix vs. no prefix?

Comment (PattaFeuFeu) [reply bigbone/src/main/kotlin/social/bigbone/api/method/admin/AdminMeasuresMethods.kt:14]: About the singular VS plural. I would rather name it `AdminMeasuresMethods` as it exposes the `/admin/measures` methods.

Comment (bocops) [reply bigbone/src/main/kotlin/social/bigbone/api/method/admin/AdminMeasuresMethods.kt:14]: FWIW, the discussion André mentioned happened in #94.

Comment (andregasser) [reply bigbone/src/main/kotlin/social/bigbone/api/method/admin/AdminMeasuresMethods.kt]: Personally, I don't like class names like `DomainBlocksMethods`, `ConversationsMethods`, etc that much, as they do not go off the tongue as easily as their colleagues `DomainBlockMethods` or `ConversationMethods`. Latter seems more natural to me. But yes... We can vote, an it seems you guys want to change it. So feel free to do it. 👍

Comment (PattaFeuFeu) [reply bigbone/src/main/kotlin/social/bigbone/api/method/admin/AdminMeasuresMethods.kt]: > not go off the tongue as easily

I never need to actually speak them, so I don’t mind that. 😁 

I’d prefer naming the methods classes like the endpoints are named, but we’re close enough so I also wouldn’t mind it too much if we kept it as-is.

**Review by andregasser — DISMISSED** [review]
👍

**Comment by PattaFeuFeu** [comment]
@andregasser Could you please approve again and merge this one? The recent merge of the main branch into this one dismissed your review.

**Review by andregasser — APPROVED** [review]
👍

**Pull request merged**